OpenRA.Mods.Common/Widgets/Logic/ReplayUtils.cs

```diff
@@ -0,0 +1,58 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using OpenRA.FileFormats;
+
+namespace OpenRA.Mods.Common.Widgets.Logic
+{
+	public static class ReplayUtils
+	{
+		static readonly Action DoNothing = () => { };
+
+		public static bool CheckReplayCompatibility(ReplayMetadata replayMeta, Action onCancel = null)
```

Comment:
Also thought about that. It does not only check that though, but also displays popup windows.